bin/spectacle/init.js

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const prompts = require('prompts');
+
+const mdRegex = /(\.mdx$|\.md$)/gm;
+const jsRegex = /(\.json$|\.js$)/gm;
+const jsxRegex = /(\.jsx$|\.js$)/gm;
+const { pathExists } = require('fs-extra');
+
+const questions = [
+  {
+    type: 'select',
+    name: 'action',
+    message: 'Create a live server or build path?',
+    choices: [
+      { title: 'Server', value: 'server' },
+      { title: 'Build', value: 'build' }
+    ]
+  },
+  {
+    type: prev => (prev === 'server' ? 'number' : null),
+    name: 'port',
+    message: 'What port for the live server?',
+    validate: val => {
+      const numLength = val.toString().length;
+      // eslint-disable-next-line no-magic-numbers
+      if (numLength > 0) {
```

Comment:
Let's keep the `val` as an actual number and assert it is greater than `0`. Right now, someone could pass in `-123` and it would pass this string length check yet not be a valid port number (which must be positive).